frontend/packages/db/supabase/migrations/20250403161902_add_suggestion_to_review_issue.sql

```diff
@@ -0,0 +1,3 @@
+-- Add suggestion column to ReviewIssue table
+ALTER TABLE "public"."ReviewIssue"
+ADD COLUMN "suggestion" text;
```

Comment:
nullable is intended?

Reply:
Oh, it was because I thought that `suggestions` would no longer be needed for `positive` issues (feedback).🙏

Reply:
Corrected the suggestion column to NOT NULL!🙏
https://github.com/liam-hq/liam/pull/1105/commits/b61ca9e6e2b93cc68013c63fe46ca4e76c5aacec